cfgov/v1/documents.py

```diff
@@ -38,6 +38,7 @@ class FilterablePagesDocument(Document):
     initial_filing_date = fields.DateField()
     model_class = fields.KeywordField()
     content = fields.TextField()
+    preview_title = fields.TextField()
```

Comment:
If we're adding `preview_title`, what do you think about adding `preview_subheading` so that we cover all of [the preview fields](https://github.com/cfpb/consumerfinance.gov/blob/7857b728bbd791264f06a932a8c1280c82d89a3e/cfgov/v1/models/learn_page.py#L42-L46)?

Reply:
@willbarton since you're out this morning, I'm pushing a commit that does this.